### ISSUE TITLE
Disable sync; Can't compile on my own

### ISSUE THREAD

**Issue by nickrupert7** [ChrisStayte/ArcMap-To-Google-Earth#4]
I'm running ArcGIS 10.3.1, and I'd like to use your tool, but unfortunately the .3 second refresh interval combined with Google Earth's tendency to "bounce" each time it refreshes make the tool somewhat inconvenient. I've found a workaround where I can disable syncing when I want to, but obviously this is not how the tool is intended to be used.
I downloaded the source code for both 10.3 and 10.2 and each time tried editing the `refreshMode` to `onChange`, but I was met with a myriad of problems.
First, the zipped/tar.gz source code for the different versions all apparently identical, because the config file for each one says `version="10.4"`, even for the zip files that are supposed to contain source for 10.3 and 10.2. I tried editing the version to 10.3 in the config file, but other problems still persisted. Next, the Carto and Geometry packages (but nothing else) are not found, so I simply delete the references and add them back in. Next, the project fails to compile unless I change all instances of `$(MSBuildExtensionsPath)\ESRI\ESRI.ArcGIS.AddIns.targets` to `$(MSBuildExtensionsPath)\ESRI\ESRI.ArcGIS.AddIns.11.targets`. Done. At this point, the tool will compile, but when I add it to ArcGIS, it doesn't register, and doesn't show up as an option for me to add the button to one of my toolbars. So I looked back at the compile warnings, and it suggests that I change `language="CLR"` to `language="CLR4.5" in the `AddIn` tag in the config file. So I did this, and now the tool compiles and properly registers to ArcGIS so I can add it to a toolbar, but it instantly crashes ArcGIS when I try to open it!

In short, I think everything I had to change to make the project compile and register with ArcGIS properly causes it to not actually work. All I REALLY need is an addin that will work with 10.3.1 and doesn't refresh on Google Earth unless my location in ArcMap changes (that is, `onChange`, rather than `onInterval`). This would help me immensely with my work.

Thanks!

P.S. I should mention that I'm doing all of this in Visual Studio 2013 targeting .Net Framework 4.5

**Comment by chrisstayte** [comment]
Based upon your description the problem looks to be from targeting .Net 4.5. Change this to 3.5 and compile. I found this from this page on supported .Net versions. http://desktop.arcgis.com/en/arcmap/10.3/get-started/system-requirements/arcobjects-sdk-system-requirements.htm

**Comment by nickrupert7** [comment]
That actually did it! Thanks!
